metadata.py

```diff
@@ -0,0 +1,31 @@
+import pandas as pd
+import os
```

Comment:
nit: You may want to replace `os` commands with `pathlib`, which is a bit modern and versatile: https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module